stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -47,12 +47,23 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
     protected val _paymentMethods = MutableLiveData<List<PaymentMethod>>()
     internal val paymentMethods: LiveData<List<PaymentMethod>> = _paymentMethods
 
+    /**
+     * Request to retrieve the value from the repository happens when initialize any fragment
+     * and any fragment will re-update when the result comes back.
+     * Represents the user last selects add or buy on the PaymentOptions/PaymentSheet, and
+     * saved/restored from the preferences.
+     */
     private val _savedSelection = MutableLiveData<SavedSelection>()
     private val savedSelection: LiveData<SavedSelection> = _savedSelection
 
     private val _transition = MutableLiveData<TransitionTargetType?>(null)
     internal val transition: LiveData<TransitionTargetType?> = _transition
 
+    /**
+     * On [com.stripe.android.paymentsheet.BaseAddCardFragment] this is set every time the add
+     * card is determined to be valid (not necessarily selected)
+     * On [com.stripe.android.paymentsheet.BasePaymentMethodsListFragment] this is set when a user selects one of the options
```

Comment:
I don't think you need to use the full package name, you should be able to just use `[BaseAddCardFragment]`. You might need to explicitly add `com.stripe.android.paymentsheet.BaseAddCardFragment` as an import.

Reply:
yep, got it.